### ISSUE TITLE
CLaMP: Contrastive Language-Music Pre-training for Cross-Modal Symbolic Music Information Retrieval

### ISSUE THREAD

**Issue by InabaTatsuro** [InabaTatsuro/papers#58]
# CLaMP: Contrastive Language-Music Pre-training for Cross-Modal Symbolic Music Information Retrieval [[ISMIR23](https://arxiv.org/pdf/2304.11029.pdf)]

<img src="https://github.com/InabaTatsuro/papers/assets/102784221/d5472ea8-0a47-4ba1-a049-c61e0143eeef" width="400">

## Abst
- trained text and symbolic music(ABC notation) with contrastive loss
- text dropout(data augmentation), bar patching(compound characters), masked music model pretraining(mask, shuffle, unchanged).

### text dropout
<img src="https://github.com/InabaTatsuro/papers/assets/102784221/b888e9ba-ae44-439c-9929-34ec167396b9" width="600">

### Masked music modeling
<img src="https://github.com/InabaTatsuro/papers/assets/102784221/7b4dd9cf-b551-4077-9aae-2526f451aee2" width="400">

### Inference
<img src="https://github.com/InabaTatsuro/papers/assets/102784221/7430cbde-3df8-4442-a859-ddf11d333390" width="600">

## memo
- what is the result if we change the encoder(M3) to MusicBERT?
-
